optimum/intel/openvino/quantization.py

```diff
@@ -332,8 +336,9 @@ def _quantize_torchmodel(
         self._set_task()
         save_directory = Path(save_directory)
         save_directory.mkdir(parents=True, exist_ok=True)
-        file_name = file_name if file_name is not None else OV_XML_FILE_NAME
-        output_path = save_directory.joinpath(file_name)
+        ov_file_name = file_name if file_name is not None else OV_XML_FILE_NAME
+        onnx_file_name = Path(file_name).with_suffix(".onnx") if file_name is not None else ONNX_WEIGHTS_NAME
```

Comment:
The following code saves `openvino_model.onnx` and fails to load `model.onnx`:
```
model_id='databricks/dolly-v2-3b'
model = AutoModelForCausalLM.from_pretrained(model_id, use_cache=use_pkv, trust_remote_code=True, torchscript=True, use_auth_token=True)
quantizer = OVQuantizer.from_pretrained(model)
quantizer.quantize(save_directory=ir_cache_dir, weights_only=True)
```

Reply:
fixed, thanks